### ISSUE TITLE
Project still active?

### ISSUE THREAD

**Issue by bedrijfsportaal** [Ultiweb-nl/factuursturen-php-client#1]
Hi,

Is this project still active? Would love to use it for one of my hobby projects! :-)

**Comment by remcom** [comment]
Yes, But didn't had the time to finalize everything. This fall it will be finalized i hope. If you want i can add you to help?

**Comment by bedrijfsportaal** [comment]
I'm a basic user but if you need help i'm here :-)

**Comment by timurgt** [comment]
Hi, is het nog gelukt met dit project?
